spring-cloud-dataflow-docs/src/main/asciidoc/task-developer-guide.adoc

```diff
@@ -57,118 +57,233 @@ If you have debug mode turned on, you can get even more information, in a line s
 
 To build your own timestamp task, follow each of these procedures:
 
-. <<task-dev-guide-building-timestamp-task-setting-up-environment>>
-. <<task-dev-guide-building-timestamp-task-getting-started>>
-. <<task-dev-guide-building-timestamp-task-making-boot-task>>
+. <<task-dev-guide-building-timestamp-task-from-initializer>>
 . <<task-dev-guide-building-timestamp-task-recording-error>>
 . <<task-dev-guide-building-timestamp-task-adding-pre-post-processing>>
 . <<task-dev-guide-building-timestamp-task-bonus-adding-mysql>>
+. <<task-dev-guide-building-timestamp-adding-tasks-to-dataflow>>
 
-[[task-dev-guide-building-timestamp-task-setting-up-environment]]
-=== Setting up the Environment
+[[task-dev-guide-building-timestamp-task-from-initializer]]
+== Developing Your Timestamp Application
 
-Before we can do anything else, we have to set up our work environment. First, we must install RabbitMQ, as described https://github.com/cppwfs/DNDataflow/blob/master/labs/InstallRabbitnMQ.pdf[in this document].
-Then we need to create a directory and add a starter project (which we created to simplify this setup). To do so:
+So now that we've used a pre-built timestamp task application, let's create one of our own.
+Most IDEs have good support for Apache Maven, so we use it as the build tool for this project.
```

Comment:
Without looking at the stream section of the docs...is that also Maven centric or does it have Gradle instructions as well?

Reply:
There is no need to have any content about maven or grade, initialzr can create maven or gradle, up to the user.  Any IDE will successfully import either a gradle or maven project.  We can show the user wants.  Also, I think it is filler to have someone check that they have installed java and show `java -version`.

Reply:
section was removed.